js/error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+window.error = (function () {
+  var errorTemplate = document.querySelector('#error').content.querySelector('.error');
+  var errorMessageElement = errorTemplate.cloneNode(true);
+  var errorButton = errorMessageElement.querySelector('.error__button');
+  var errorText = errorMessageElement.querySelector('.error__message');
+
+  return {
+    errorHandler: function (error, errBtnHandler) {
+      errorText.innerText = 'Sending error: ' + error;
+      errorButton.addEventListener('click', errBtnHandler);
+      document.querySelector('body').appendChild(errorMessageElement);
+      var errorElement = document.querySelector('body').querySelector('.error');
```

Comment:
errorElement is the same as errorMessageElement

Reply:
not the same as errorMessageElement is inside the variable which is not added to the body yet,
where errorElement is selected from the body after adding errorMessageElement to the DOM

Reply:
You're querying the same element after you've appended it to the body. It is not optimal. Don't see how it is different. 
line 12 `document.querySelector('body').appendChild(errorMessageElement);`
line 12 `document.querySelector('body').querySelector('.error');`